zenoh/src/queryable.rs

```diff
@@ -100,17 +104,96 @@ impl Query {
     pub fn attachment(&self) -> Option<&Attachment> {
         self.inner.attachment.as_ref()
     }
+    /// Sends a reply in the form of [`Sample`] to this Query.
+    ///
+    /// By default, queries only accept replies whose key expression intersects with the query's.
+    /// Unless the query has enabled disjoint replies (you can check this through [`Query::accepts_replies`]),
+    /// replying on a disjoint key expression will result in an error when resolving the reply.
+    #[inline(always)]
```

Comment:
This function should be marked as unstable and only for internal usage really.

Reply:
It is being used by Query.reply(...) in zenoh-backend-traits, which does not require "unstable" feature of zenoh.

Reply:
Then I believe we should add the "unstable" feature to the `zenoh-backend-traits`